app/views/residents/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<p id="notice"><%= notice %></p>
```

Comment:
`app/views/residents/index.html.erb` and `app/views/houses/residents.html.erb` could use a shared partial